### ISSUE TITLE
Add currencies missing in Admin

### ISSUE THREAD

**Issue by rstevens354** [medusajs/medusa#3215]
### Blank page when trying to add new currencies

In Medusa Admin Starter im getting an empty list of currencies (Settings > Currencies) and a blank page (modal) when clicking the Add Currencies button

### System information

@medusajs/medusa": "^1.6.0"
@medusajs/medusa-cli": "^1.3.3"
Node.js version: v16.14.2
Database: Postgres
Operating system: Windows
Browser (if relevant): Chrome Version 110.0.5481.78 (Official Build) (64-bit)

### Steps to reproduce the behavior

1. In Medusa Admin go to **Settings**
2. Click on **Currencies**
3. Here the list of **Store Currencies** is blank

![Screenshot 2023-02-09 080248](https://user-images.githubusercontent.com/9464748/217755262-5667c3b4-de0b-4faf-87e0-39cda373ed60.png)

4.Click on **Edit Currencies** - i can see a populated list

![Screenshot 2023-02-09 080338](https://user-images.githubusercontent.com/9464748/217755708-00ae8b7f-cd9a-4e5a-a996-f8ce8fef3060.png)

5. Click on **Add Currencies** and i get a blank modal

![Screenshot 2023-02-09 080401](https://user-images.githubusercontent.com/9464748/217755905-d25a535a-cac0-4653-a92a-51f3b5af5dc5.png)

6. Unsure if this is related, but im getting this error in Console

![image](https://user-images.githubusercontent.com/9464748/217756350-5d6b38bd-6977-410d-a396-ce1aa3e2a50e.png)

### Expected behavior

Ability to add, view and edit currencies

**Comment by olivermrbl** [comment]
I am unable to reproduce your issue. Can I get you to provide a reprository? :)

**Comment by rstevens354** [comment]
It’s the starter as described here
https://docs.medusajs.com/usage/create-medusa-app

`npx create-medusa-app`

I did npm instead or yarn - could that be a factor or maybe a problem with my Postgres? 

I seeded with npm run seed?

**Comment by olivermrbl** [comment]
Hmm, can I get you to check what version of Medusa is installed in your `yarn.lock` or `package-lock.json`?

**Comment by rstevens354** [comment]
```
"dependencies": {
        "@medusajs/medusa": "^1.6.0",
        "@medusajs/medusa-cli": "^1.3.3",
        "medusa-fulfillment-manual": "^1.1.31",
        "medusa-interfaces": "^1.3.3",
        "medusa-payment-manual": "^1.0.16",
        "medusa-payment-stripe": "^1.1.45",
        "typeorm": "^0.2.36"
      },
```
      
   and 
   
```
   "node_modules/@medusajs/medusa": {
      "version": "1.7.5",
```

**Comment by olivermrbl** [comment]
Can I get you to verify that the request to retrieve currencies is being fired? It happens when you click the "Add Currencies" button in the modal.

The request should look something like this:
```
GET .../admin/currencies?limit=15&offset=0 
```

**Comment by rstevens354** [comment]
Yep the request fires, but then almost immediately i get a failed request

"GET /admin/currencies?limit=15&offset=0 HTTP/1.1" 304 - "http://localhost:7000/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36"

"GET /admin/batch-jobs?failed_at=null HTTP/1.1" 304 - "http://localhost:7000/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36"

**Comment by kasperkristensen** [comment]
@rstevens354 the `GET /admin/batch-jobs?failed_at=null` is unrelated, it is simply polling for any batch jobs that might be running. I can't remember when we added the endpoint to retrieve currencies, but it was definitely later than 1.6.0. The admin project is always intended to be used with the latest version of `@medusajs/medusa`, and your issue is most likely due to you running an old version of Medusa on your server. I would suggest that you update your server to the latest version of Medusa and see if fixes the issue 😄

**Comment by olivermrbl** [comment]
Closing as we are unable to reproduce with the details provided.

**Comment by rktyt** [comment]
Same issue for me.

node: v18.13.0
npm: v8.19.3
OS: MacOS 13.2.1 (arm64)

```shell-session
$ npx create-medusa-app
✔ Where should your project be installed? · medusa
✔ Which Medusa starter would you like to install? · medusa-starter-default
✔ Which storefront starter would you like to install? · Next.js Starter
⠸ Creating new project from git: https://github.com/medusajs/medusa-starter-default.git
✔ Created starter directory layout

⠏ Installing packages...
✔ Packages installed
Initialising git in medusa/backend
Create initial git commit in medusa/backend
⠋ Seeding database
> medusa-starter-default@0.0.1 seed
> medusa seed -f ./data/seed.json

[medusa-config] ⚠️ redis_url not found. A fake redis instance will be used.
info:    Using fake Redis
warn:    You don't have any notification provider plugins installed. You may want to add one to your project.
warn:    You don't have any taxProviders provider plugins installed. You may want to add one to your project.
warn:    No search engine provider was found: make sure to include a search plugin to enable searching
⠦ Seeding database
✔ Seed completed
⠇ Creating new project from git: https://github.com/medusajs/nextjs-starter-medusa.git
✔ Created starter directory layout

⠧ Installing packages...
✔ Packages installed
Initialising git in medusa/storefront
Create initial git commit in medusa/storefront
⠼ Creating new project from git: https://github.com/medusajs/admin.git
✔ Created starter directory layout

⠧ Installing packages...
✔ Packages installed
Initialising git in medusa/admin
Create initial git commit in medusa/admin

  Your project is ready 🚀. The available commands are:
  
    Medusa API
    cd medusa/backend
    yarn start

    Admin
    cd medusa/admin
    yarn start
  

    Storefront
    cd medusa/storefront
    yarn start
```

After installation, `node_modules` and `package-lock.json` were removed and dependent packages were updated.
*Some packages cannot be brought up to date.

`backend/package.json`
```json
{
  "name": "medusa-starter-default",
  "version": "0.0.1",
  "description": "A starter for Medusa projects.",
  "author": "Sebastian Rindom <skrindom@gmail.com>",
  "license": "MIT",
  "scripts": {
    "seed": "medusa seed -f ./data/seed.json",
    "build": "babel src -d dist --extensions \".ts,.js\"",
    "start": "medusa develop"
  },
  "dependencies": {
    "@medusajs/medusa": "^1.7.7",
    "@medusajs/medusa-cli": "^1.3.8",
    "medusa-fulfillment-manual": "^1.1.36",
    "medusa-interfaces": "1.3.6",
    "medusa-payment-manual": "^1.0.22",
    "medusa-payment-stripe": "^1.1.53",
    "typeorm": "0.2.45"
  },
  "repository": "https://github.com/medusajs/medusa-starter-default.git",
  "keywords": [
    "sqlite",
    "ecommerce",
    "headless",
    "medusa"
  ],
  "devDependencies": {
    "@babel/cli": "^7.20.7",
    "@babel/core": "^7.20.12",
    "@babel/preset-typescript": "^7.18.6",
    "babel-preset-medusa-package": "^1.1.19"
  }
}
```

`admin/package.json`
```json
{
  "name": "medusa-admin",
  "private": true,
  "description": "A proprietary Admin system to manage Medusa Stores",
  "version": "0.1.0",
  "author": "@medusajs",
  "resolutions": {
    "devcert": "1.2.1",
    "postcss": "^8.4.5",
    "@types/react": "^18.0.21"
  },
  "dependencies": {
    "@hookform/error-message": "^2.0.1",
    "@medusajs/medusa": "^1.7.7",
    "@radix-ui/react-accordion": "^1.1.0",
    "@radix-ui/react-avatar": "^1.0.1",
    "@radix-ui/react-collapsible": "^1.0.1",
    "@radix-ui/react-dialog": "^1.0.2",
    "@radix-ui/react-dropdown-menu": "^2.0.2",
    "@radix-ui/react-popover": "^1.0.3",
    "@radix-ui/react-portal": "^1.0.1",
    "@radix-ui/react-radio-group": "^1.1.1",
    "@radix-ui/react-select": "^1.2.0",
    "@radix-ui/react-switch": "^1.0.1",
    "@radix-ui/react-tooltip": "^1.0.3",
    "@segment/analytics-next": "^1.51.0",
    "@tailwindcss/line-clamp": "^0.4.2",
    "@tanstack/react-query": "^4.24.6",
    "axios": "^1.3.3",
    "clsx": "^1.2.1",
    "confetti-js": "^0.0.18",
    "copy-to-clipboard": "^3.3.3",
    "crypto-browserify": "^3.12.0",
    "emoji-picker-react": "^4.4.7",
    "framer-motion": "^9.0.4",
    "immutability-helper": "^3.1.1",
    "iso8601-duration": "^2.1.1",
    "lodash": "^4.17.21",
    "medusa-react": "^4.0.4",
    "moment": "^2.29.4",
    "query-string": "^8.1.0",
    "randomatic": "^3.1.1",
    "react": "^18.2.0",
    "react-collapsible": "^2.10.0",
    "react-country-flag": "^3.0.2",
    "react-currency-input-field": "^3.6.10",
    "react-datepicker": "^4.10.0",
    "react-dnd": "^16.0.1",
    "react-dnd-html5-backend": "^16.0.1",
    "react-dom": "^18.2.0",
    "react-helmet": "^6.1.0",
    "react-highlight-words": "^0.20.0",
    "react-hook-form": "^7.43.1",
    "react-hot-toast": "^2.4.0",
    "react-hotkeys-hook": "^4.3.5",
    "react-json-tree": "^0.18.0",
    "react-jwt": "^1.1.8",
    "react-router-dom": "^6.8.1",
    "react-select": "^5.7.0",
    "react-table": "^7.8.0",
    "stream-browserify": "^3.0.0",
    "type-fest": "^3.6.0",
    "uuid": "^9.0.0"
  },
  "devDependencies": {
    "@babel/core": "^7.20.12",
    "@storybook/addon-essentials": "^6.5.16",
    "@storybook/builder-vite": "^0.4.2",
    "@storybook/react": "^6.5.16",
    "@tailwindcss/forms": "^0.5.3",
    "@tailwindcss/line-clamp": "^0.4.2",
    "@types/randomatic": "^3.1.3",
    "@types/react": "^18.0.28",
    "@types/react-datepicker": "^4.8.0",
    "@types/react-dom": "^18.0.11",
    "@types/react-highlight-words": "^0.16.4",
    "@types/react-table": "^7.7.14",
    "@types/uuid": "^9.0.0",
    "@typescript-eslint/eslint-plugin": "^5.52.0",
    "@typescript-eslint/parser": "^5.52.0",
    "@vitejs/plugin-react": "^3.1.0",
    "autoprefixer": "^10.4.13",
    "babel-loader": "^9.1.2",
    "chromatic": "^6.17.0",
    "eslint": "^8.34.0",
    "eslint-config-google": "^0.14.0",
    "eslint-config-prettier": "^8.6.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-unused-imports": "^2.0.0",
    "postcss": "^8.4.21",
    "prettier": "^2.8.4",
    "storybook-addon-state": "^1.0.3",
    "tailwindcss": "^3.2.7",
    "tailwindcss-radix": "^2.7.0",
    "ts-node": "^10.9.1",
    "typescript": "^4.9.5",
    "vite": "^4.1.1"
  },
  "scripts": {
    "dev": "vite --port 7000",
    "develop": "npm run dev",
    "check": "tsc",
    "build": "vite build",
    "preview": "vite preview --port 7000",
    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json,md}\"",
    "start": "npm run dev",
    "serve": "npm run preview",
    "test": "echo \"Write tests! && exit 1",
    "storybook": "start-storybook -p 6006",
    "build-storybook": "build-storybook"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/medusajs/admin"
  },
  "bugs": {
    "url": "https://github.com/medusajs/admin/issues"
  }
}
```

**Comment by rktyt** [comment]
I found out that it is a CSS problem in admin.

The class of the content of the dialog is `inter-base-regular h-full transition-transform translate-x-full flex flex-col justify-between duration-200 translate-x-0 `but the `translate-x-full` style is applied instead of the `translate-x-0` style.

----

For Demo (https://demo.medusajs.com/a/settings/currencies)
![スクリーンショット 2023-02-17 17 44 09](https://user-images.githubusercontent.com/26789317/219596479-60927319-bbd6-448a-803f-2f5b2ef33c41.png)

`translate-x-0` is applied.

----

Created by create-medusa-app

![スクリーンショット 2023-02-17 17 47 37](https://user-images.githubusercontent.com/26789317/219597290-43633de9-f5f1-4132-8f1b-fe00f77b71fb.png)

`translate-x-full` is applied.

*This is true even if you do not update admin's dependent packages

----

If I get more details, I'll add them.

**Comment by daflav** [comment]
same bug

**Comment by daflav** [comment]
Same bug also in edit variant :
![image](https://user-images.githubusercontent.com/9709182/220129436-66454b48-c91a-424e-8e0b-63086fd7d9b7.png)

**Comment by chemicalkosek** [comment]
Something changed in tailwind 3.2.5 <https://github.com/tailwindlabs/tailwindcss/issues/10603#issuecomment-1433743390>

**Comment by olivermrbl** [comment]
Solved in #897
